app/views/users/show.html.erb

```diff
@@ -1 +1,23 @@
-<h1>This is the Users#shows page.</h1>
+  <div class="user_info">
+   <%= image_tag ("#{@user.photo}"), alt: "#{@user.name}'s photo", width: "100px", height: "80px" %>
+   <span class="text_info">
+   <span class="user_name"><%= "Name: #{@user.name}"%></span>
+   <br>
+   <span class="numbers"><%= "Number of posts: #{@user.posts_counter}"%></span>
+   </span>
+   <br>
+  </div>
+
+<div class="bio_container">
+  <p class="bio_heading">Bio</p><br>
+  <%= @user.bio%><br><br>
+</div>
+<% @user_posts.each do |post|%>
+  <div class="post_container">
+  <strong><%= "Post ##{post.id}"%></strong> <br> 
+  <p><%= post.text %></p>
+  <span class="post_details"><%= "Comments: #{post.comments_counter}  Likes: #{post.likes_counter}"%></span> <br><br>
+  </div>
```

Comment:
- [ ] Use `three_most_recent` method to render the user's 3 most recent post.

Reply:
Thank you for the suggestion. I have worked on it.